patroni/ha.py

```diff
@@ -890,6 +891,13 @@ def _is_healthiest_node(self, members: Collection[Member], check_replication_lag
                                 or not self.cluster.sync.leader_matches(st.member.name):
                             return False
                         logger.info('Ignoring the former leader being ahead of us')
+                    if my_wal_position == st.wal_position and self.patroni.failover_priority < priority:
+                        # There's a higher priority non-lagging replica
+                        logger.info(
+                            '%s is not lagging and has a higher priority',
+                            st.member.name
+                        )
```

Comment:
It perfectly fits to 120 characters.
```suggestion
                        logger.info('%s is not lagging and has a higher priority', st.member.name)
```

Also, maybe add actual priority values to the log.